### PR TITLE
feat(ingress): remove nginx proxy sidecar

### PR DESCRIPTION
**Pull request by VincentGuoCQ**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes # http://jira.caicloud.xyz/browse/CPS-7327

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @hicaistar 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -98,8 +96,6 @@ func (c *Configuration) AddFlags(fs *pflag.FlagSet) {
 
 	fs.Var(&c.AdditionalTolerations, "additional-tolerations", "A comma separated list of k8s `TolerationKeys`")
 
-	fs.StringVar(&c.Proxies.Sidecar, "proxy-sidecar", defaultIngressSidecarImage, "`Image` of ingress controller sidecar")
```
Comment (hanxueluo) [inline pkg/config/config.go:101]: 参数保留着。类似 default-http-backend 的处理。

**Comment by hanxueluo** [comment]
depends on #921

**Comment by hicaistar** [comment]
LGTM

**Pull request merged**